### PR TITLE
shagrath.PS3MediaServer version 1.90.1

### PR DESCRIPTION
**Pull request by JoaquinPro64**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/53972)

### CODE REVIEW

File: manifests/s/shagrath/PS3MediaServer/1.90.1/shagrath.PS3MediaServer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: shagrath.PS3MediaServer
+PackageVersion: 1.90.1
+PackageLocale: en-US
+Publisher: shagrath
```
Comment (ItzLevvie) [inline manifests/s/shagrath/PS3MediaServer/1.90.1/shagrath.PS3MediaServer.locale.en-US.yaml:7]:
```suggestion
Publisher: PS3 Media Server
```
This must match Control Panel.

File: manifests/s/shagrath/PS3MediaServer/1.90.1/shagrath.PS3MediaServer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: shagrath.PS3MediaServer
+PackageVersion: 1.90.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: http://sourceforge.net/projects/ps3mediaserver/files/pms-1.90.1-setup-full-x64.exe/download
```
Comment (ItzLevvie) [inline manifests/s/shagrath/PS3MediaServer/1.90.1/shagrath.PS3MediaServer.installer.yaml:9]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/ps3mediaserver/files/pms-1.90.1-setup-full-x64.exe/download
```

File: manifests/s/shagrath/PS3MediaServer/1.90.1/shagrath.PS3MediaServer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: shagrath.PS3MediaServer
+PackageVersion: 1.90.1
+PackageLocale: en-US
+Publisher: shagrath
+PackageName: PS3 Media Server
+License: GNU General Public License
```
Comment (ItzLevvie) [inline manifests/s/shagrath/PS3MediaServer/1.90.1/shagrath.PS3MediaServer.locale.en-US.yaml:9]:
```suggestion
License: GPL-2.0-only
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**